spec/game_checker_spec.rb

```diff
@@ -0,0 +1,72 @@
+require "rspec"
+
+require_relative "../lib/game_checker"
+require_relative "../lib/board"
+
+describe "GameChecker" do
+  subject(:checker) { GameChecker.new }
+  let(:board) { Board.new(checker) }
+  let(:player1) { TestPlayer.new("Z") }
```

Comment:
Why do you initialize the board with a checker?  Board just needs to be passed into the GameChecker methods and you can pass any mock board into these methods.  The mock board methods could just return booleans.

Reply:
I think I'm just gonna move GameChecker to be another arg for Game. There's no real reason for it to be inside Board at all, bc all the functions take Board as an arguments anyway. 

Also just using a lot of stubs for TestBoard, needs to handle negative cases too.

Reply:
Okay